getting_started/step_by_step/signals.rst

```diff
@@ -371,6 +476,17 @@ To emit values along with the signal, add them as extra arguments to the
         health -= amount
         emit_signal("health_changed", old_health, health)
 
+ .. code-tab:: csharp C#
+
+    public void TakeDamage(int amount)
+    {
+        _health -= amount;
+
+        if (_health <= 0)
+        {
+            EmitSignal(nameof(HealthDepletedEventHandler));
+        }
```

Comment:
This C# example doesn't match the GDScript version. This example corresponds to the GDScript example that ends at line 437.